### PR TITLE
Add in some simple diagnostics and install manpages

### PR DESCRIPTION
**Pull request by MikeFultonDev**

### CODE REVIEW

File: buildenv [hunk 1]
```diff
@@ -8,6 +8,9 @@ GIT_VERSION="2.39.2"
 export ZOPEN_GIT_URL="https://github.com/git/git"
 export ZOPEN_GIT_DEPS="curl git make m4 perl autoconf automake help2man texinfo xz zlib openssl expat gettext coreutils diffutils bash"
 
+export ZOPEN_TARBALL_MAN_REL="2.9.5"
```
Comment (IgorTodorovskiIBM) [inline buildenv:11]: Should this be 2.39.2?

File: buildenv [hunk 2]
```diff
@@ -118,7 +125,40 @@ EOF
 zopen_post_install()
 {
   # Install a cacert.pem to be used (optionally) by the customer
-  zopen update-cacert -f $1
+  if ! zopen update-cacert -f $1 ; then
+    return 4
```
Comment (IgorTodorovskiIBM) [inline buildenv:129]: I don't think we check the return code for post-install - https://github.com/ZOSOpenTools/meta/blob/main/bin/lib/zopen-build#L1339. Maybe we should? You could also call the `printError` function

File: buildenv (hunk 1 above)
Comment (MikeFultonDev) [reply buildenv:11]: I don’t know why but the doc versions aren’t the same as the code versions. That’s the latest one.

File: buildenv (hunk 2 above)
Comment (MikeFultonDev) [reply buildenv:129]: Will do. Yeah - we should check the return code.

File: buildenv (hunk 1 above)
Comment (MikeFultonDev) [reply buildenv:11]: added a block comment to explain versioning

File: buildenv (hunk 2 above)
Comment (MikeFultonDev) [reply buildenv:137]: added printSoftError so that code continues and the caller can 'fail out' if necessary

Comment (MikeFultonDev) [reply buildenv:137]: added check to post_install in the manpagetweak PR: https://github.com/ZOSOpenTools/meta/pull/239

File: buildenv (hunk 1 above)
Comment (IgorTodorovskiIBM) [reply buildenv:11]: Hmm, could it a result of the sorting?
```
git-manpages-2.39.2.tar.xz                        14-Feb-2023 17:56    544K
vs
git-manpages-2.9.5.tar.xz                          10-Aug-2017 17:58    383K
```

Comment (MikeFultonDev) [reply buildenv]: fixed to use the proper level of manpages.

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**